components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml

```diff
@@ -49,8 +49,16 @@ paths:
                 $ref: '#/components/schemas/Error'
         '401':
           description: Unauthorized
+          content:
```

Comment:
lets remove content for 401, 403 reason: https://github.com/wso2/product-is/issues/6414